pkg/controller/datavolume-conditions.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2020 The CDI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+)
+
+const (
+	boundFalse      = "BoundChangeFalse"
+	transferRunning = "TransferRunning"
+	pvcBound        = "PVCBound"
+	pvcPending      = "PVCPending"
+	claimLost       = "ClaimLost"
+	notFound        = "NotFound"
```

Comment:
Not clear to me why for example `PVCPending` is preferable to `PVC is pending` for condition reason